src/main/webapp/dev/js/main/studentMotd.es6

```diff
@@ -6,7 +6,8 @@ function fetchMotd(motdUrl, motdContentSelector, motdContainerSelector) {
         type: 'GET',
         url: `${window.location.origin}/${motdUrl}`,
         success(data) {
-            $(motdContentSelector).html(data);
+            const bodyContent = data.match(/<body[^>]*>[\s\S]*<\/body>/gi);
```

Comment:
@thenaesh I know `.` is used to match all character but due to some reason, it doesn't seem to work. I checked the regular expression on [Regexr](http://regexr.com/) the `.` notation regular expression isn't working while the `\s\S` works fine.

_[Regex matching screenshot]_: `<body[^>]*>[\s\S]*<\/body>`
![screenshot from 2017-07-21 14-28-59](https://user-images.githubusercontent.com/9383897/28456796-2768b844-6e21-11e7-9ac0-d7402bf6c318.png)

_[Not matching Regex]_: `<body[^>]*>.*<\/body>`
![screenshot from 2017-07-21 14-33-04](https://user-images.githubusercontent.com/9383897/28456881-8d13dd04-6e21-11e7-8dcf-43d03f9cde40.png)